test/integration/Governance/categorizeProposal.js

```diff
@@ -0,0 +1,35 @@
+const { expect } = require('chai');
+const { proposalTitle, proposalSD, proposalDescHash } = require('./proposalFixture');
+
+describe('categorizeProposal', function () {
+  let proposalId;
+  beforeEach(async function () {
+    const { gv: governance } = this.contracts;
+    const categoryId = 0;
+    const [member] = this.accounts.members;
+    proposalId = await governance.getProposalLength();
+
+    await governance.connect(member).createProposal(proposalTitle, proposalSD, proposalDescHash, categoryId);
+  });
+
+  it('should fail to categorize proposal if sender is not owner', async function () {
```

Comment:
There's no `owner` anymore. The current check is whether the current role has rights.